src/main/java/frc/robot/commands/MoveToSelectedPose.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.controls.Operator;
+import frc.robot.subsystems.Drivetrain;
+
+/**
+ * Uses PID to make the robot go to the selected pose
+ */
+public class MoveToSelectedPose extends CommandBase {
+
+  private final Drivetrain m_drive;
+  
+  public MoveToSelectedPose(Drivetrain drive) {
+    m_drive = drive;    
+    addRequirements(drive);
+  }
+
+  @Override
+  public void initialize(){
+    m_drive.setAllOptimize(true);
+  }
+  
+  @Override
+  public void execute() {
+    double x = Operator.selectedNode.scorePose.getX();
+    double y = Operator.selectedNode.scorePose.getY();
+    double rot = Operator.selectedNode.scorePose.getRotation().getRadians();
+
+    System.out.printf("Moving to (%.2f, %.2f) at %.2f degrees", x, y, Units.radiansToDegrees(rot));
```

Comment:
don't have this in execute this will spam the rio log when ever it is called. if you really want to print something do so in initalize and end. Even then genrealy you shouldn't print stuff on normal commands.